### DISCUSSION TITLE
MLB Scoreboard

### DISCUSSION THREAD

**Discussion by skhull2** [riffnshred/nhl-led-scoreboard#444] [General]
Would you be willing and/or able to create an MLB Scoreboard using the same process as you used with the NHL Scoreboard?  I love the initial setup wizard after you log in using Putty.  The NHL Scoreboard works great with a 64x32 matrix but the MLB one is defaulted to 32x32 & it is difficult to modify that project to work on a 64x32 matrix.

**Comment by riffnshred** [comment]
Glad you like the tool.  Sadly we are not familiar with that project at all and the moment I have time to work on this current project, I want to spend it all on it. Maybe someone on the dev team of the MLB scoreboard project could check our code and see how they could implement it ?
